ecf/scripts/stats/analyses/jevs_analyses_urma_precip_stats_vhr_master.ecf

```diff
@@ -0,0 +1,56 @@
+#PBS -N evs_analyses_urma_grid2obs_stats_vhr_%VHR%
```

Comment:
Check line 1. I think it should be `_precip_` instead of `_grid2obs_`

Reply:
Fixed.